src/domUtil.js

```diff
@@ -342,7 +342,7 @@ class DomUtil {
                     xmlRoot.appendChild(xmlElement);
                 }
                 else if (t == "object") {
-                    if (value.length !== undefined && value.length !== null) {
+                    if (value.length !== undefined && value.length !== null && value.length > 0) {
```

Comment:
What is the point of this change?

Reply:
to handle this kind of attribute:
```xml
<element label="label" name="name" >
      <attribute label="Primary key" name="id" />
</element>
```

Reply:
This function converts JSON to XML. I don't understand to what JSON your example comments or why this needs to be changed. Changes in the JSON <=> XML conversion are potentially very impactful. This would also warrant appropriate unit tests. 
Are we changing the XML created from the JSON is any way? If yes, then. this is a breaking change